spec/unit/azure_server_create_spec.rb

```diff
@@ -866,56 +866,42 @@ def test_params(testxml, chef_config, role_name, host_name)
       it "should set autoUpdateClient flag to true" do
         @server_instance.config[:auto_update_client] = true
         public_config = "{\"client_rb\":\"chef_server_url \\t \\\"https://localhost:443\\\"\\nvalidation_client_name\\t\\\"chef-validator\\\"\",\"runlist\":\"\\\"getting-started\\\"\",\"autoUpdateClient\":\"true\",\"deleteChefConfig\":\"false\",\"uninstallChefClient\":\"false\",\"custom_json_attr\":{},\"bootstrap_options\":{\"chef_server_url\":\"https://localhost:443\",\"validation_client_name\":\"chef-validator\"}}"
-
-        expect(Base64).to receive(:encode64).with(public_config)
         @server_instance.get_chef_extension_public_params
       end
 
       it "should set autoUpdateClient flag to false" do
         @server_instance.config[:auto_update_client] = false
         public_config = "{\"client_rb\":\"chef_server_url \\t \\\"https://localhost:443\\\"\\nvalidation_client_name\\t\\\"chef-validator\\\"\",\"runlist\":\"\\\"getting-started\\\"\",\"autoUpdateClient\":\"false\",\"deleteChefConfig\":\"false\",\"uninstallChefClient\":\"false\",\"custom_json_attr\":{},\"bootstrap_options\":{\"chef_server_url\":\"https://localhost:443\",\"validation_client_name\":\"chef-validator\"}}"
-
-        expect(Base64).to receive(:encode64).with(public_config)
         @server_instance.get_chef_extension_public_params
       end
 
       it "sets deleteChefConfig flag to true" do
```

Comment:
There should be an expectation that this flag gets set somewhere and it should be tested -- I don't see where that's happening, even in the old code. Similar feedback for all other config in this file.